src/transformers/models/bert/modeling_flax_bert.py

```diff
@@ -142,278 +145,240 @@ class FlaxBertEmbedding(nn.Module):
+            self.config.max_position_embeddings,
+            self.config.hidden_size,
+            initializer_range=self.config.initializer_range,
             name="position_embeddings",
             dtype=self.dtype,
-        )(jnp.atleast_2d(position_ids.astype("i4")))
-        t_emb = FlaxBertEmbedding(
-            self.type_vocab_size,
-            self.hidden_size,
-            kernel_init_scale=self.kernel_init_scale,
+        )
+        self.token_type_emb = FlaxBertEmbedding(
+            self.config.type_vocab_size,
+            self.config.hidden_size,
+            initializer_range=self.config.initializer_range,
             name="token_type_embeddings",
             dtype=self.dtype,
-        )(jnp.atleast_2d(token_type_ids.astype("i4")))
+        )
+        # fmt: off
+        self.layer_norm = FlaxBertLayerNorm(hidden_size=self.config.hidden_size, name="layer_norm", dtype=self.dtype)
+        # fmt: on
+        self.dropout = nn.Dropout(rate=self.config.hidden_dropout_prob)
+
+    def __call__(self, input_ids, token_type_ids, position_ids, attention_mask, deterministic: bool = True):
+        # Embed
+        inputs_embeds = self.word_emb(jnp.atleast_2d(input_ids.astype("i4")))
+        position_embeds = self.pos_emb(jnp.atleast_2d(position_ids.astype("i4")))
+        token_type_embeds = self.token_type_emb(jnp.atleast_2d(token_type_ids.astype("i4")))
 
         # Sum all embeddings
-        summed_emb = w_emb + jnp.broadcast_to(p_emb, w_emb.shape) + t_emb
+        hidden_states = inputs_embeds + jnp.broadcast_to(position_embeds, inputs_embeds.shape) + token_type_embeds
 
         # Layer Norm
-        layer_norm = FlaxBertLayerNorm(name="layer_norm", dtype=self.dtype)(summed_emb)
-        embeddings = nn.Dropout(rate=self.dropout_rate)(layer_norm, deterministic=deterministic)
-        return embeddings
+        hidden_states = self.layer_norm(hidden_states)
+        hidden_states = self.dropout(hidden_states, deterministic=deterministic)
+        return hidden_states
 
 
 class FlaxBertAttention(nn.Module):
-    num_heads: int
-    head_size: int
-    dropout_rate: float = 0.0
-    kernel_init_scale: float = 0.2
+    config: BertConfig
     dtype: jnp.dtype = jnp.float32  # the dtype of the computation
 
-    @nn.compact
-    def __call__(self, hidden_states, attention_mask, deterministic: bool = True):
+    def setup(self):
+        self.self_attention = nn.attention.SelfAttention(
+            num_heads=self.config.num_attention_heads,
+            qkv_features=self.config.hidden_size,
+            dropout_rate=self.config.attention_probs_dropout_prob,
+            kernel_init=jax.nn.initializers.normal(self.config.initializer_range, self.dtype),
+            bias_init=jax.nn.initializers.zeros,
+            name="self",
+            dtype=self.dtype,
+        )
+        # fmt: off
+        self.layer_norm = FlaxBertLayerNorm(hidden_size=self.config.hidden_size, name="layer_norm", dtype=self.dtype)
+        # fmt: on
```

Comment:
Why the tags? we can have 
```
self.layer_norm = FlaxBertLayerNorm(
    hidden_size=self.config.hidden_size, name="layer_norm", dtype=self.dtype
)
```
if it's too long.

Reply:
It's the copied from check :D It doesn't play well with black :-/ Any idea on how to easily fix that?

Reply:
Ugh, annoying. Will fix that script in the future to play with black better then.